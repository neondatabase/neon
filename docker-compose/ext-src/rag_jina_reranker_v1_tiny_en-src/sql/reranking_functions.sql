-- Reranking function tests
SELECT rag_jina_reranker_v1_tiny_en.rerank_distance('the cat sat on the mat', 'the baboon played with the balloon');

SELECT rag_jina_reranker_v1_tiny_en.rerank_distance('the cat sat on the mat', ARRAY['the baboon played with the balloon', 'the tanks fired at the buildings']);

SELECT rag_jina_reranker_v1_tiny_en.rerank_score('the cat sat on the mat', 'the baboon played with the balloon');

SELECT rag_jina_reranker_v1_tiny_en.rerank_score('the cat sat on the mat', ARRAY['the baboon played with the balloon', 'the tanks fired at the buildings']);