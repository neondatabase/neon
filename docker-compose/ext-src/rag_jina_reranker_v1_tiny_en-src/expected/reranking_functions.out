-- Reranking function tests
SELECT rag_jina_reranker_v1_tiny_en.rerank_distance('the cat sat on the mat', 'the baboon played with the balloon');
 rerank_distance 
-----------------
       0.8989152
(1 row)

SELECT rag_jina_reranker_v1_tiny_en.rerank_distance('the cat sat on the mat', ARRAY['the baboon played with the balloon', 'the tanks fired at the buildings']);
    rerank_distance    
-----------------------
 {0.8989152,1.3018152}
(1 row)

SELECT rag_jina_reranker_v1_tiny_en.rerank_score('the cat sat on the mat', 'the baboon played with the balloon');
 rerank_score 
--------------
   -0.8989152
(1 row)

SELECT rag_jina_reranker_v1_tiny_en.rerank_score('the cat sat on the mat', ARRAY['the baboon played with the balloon', 'the tanks fired at the buildings']);
      rerank_score       
-------------------------
 {-0.8989152,-1.3018152}
(1 row)

